OpenSim/Simulation/Model/ModelDisplay.h

```diff
@@ -0,0 +1,87 @@
+#ifndef OPENSIM_MODEL_DISPLAY_H_
+#define OPENSIM_MODEL_DISPLAY_H_
+/* -------------------------------------------------------------------------- *
+ *                             OpenSim:  ModelDisplay.h                       *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
+ * Author(s): Ayman Habib                                                     *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+
+// INCLUDE
+#include <iostream>
+#include <OpenSim/Simulation/osimSimulationDLL.h>
+#include "SimTKcommon.h"
+#include <OpenSim/Common/Property.h>
+#include <OpenSim/Simulation/Model/ModelDisplayHints.h>
+
+namespace OpenSim {
+
+class Body;
+class Model;
+
+
+//=============================================================================
+//=============================================================================
+/**
+ * A class that holds the Display Attributes (ModelDisplay) of a full Model 
+ * displayed in Visualizer. Initially these are serializable ModelDisplayHints 
+ * but in the future can be expanded to include search paths for Geometry, Texture
+ * Renderer preferences, lights, cameras etc.
+ *
+ * @author Ayman Habib
```

Comment:
I likely do not adhere to this and I should, but we should strive to describe new classes using existing abstractions and describe new concepts/abstractions introduced by the class or a logical grouping of classes. For example, what are Display Attributes? Why is a group of these in a new class?
Is a "full Model" different from a partial Model or even a ModelComponent. If so, why? If we need a special Component that manages the Appearances of Components in a Visualizer why is it a) called ModelDisplay and b) an Object and not a Component itself so it can leverage system resources (like realization Stages) to determine what can be visualized and how. E.g no forces when realized only to position (from. IK).